atomics/T1090.003/T1090.003.yaml

```diff
@@ -0,0 +1,36 @@
+attack_technique: T1090.003
+display_name: 'Proxy: Multi-hop Proxy'
+atomic_tests:
+- name: Psiphon
+  description: |
+    Psiphon 3 is a circumvention tool from Psiphon Inc. that utilizes VPN, SSH and HTTP Proxy technology to provide you with uncensored access to Internet.
+    Note that this test may conflict with pre-existing system configuration.
+  supported_platforms:
+    - windows
+  input_arguments:
+    Psiphon_file:
+      description: backup of the pre-existing system proxy configuration
```

Comment:
The description talks about the proxy backup file but the default value points to the bat file that does the installation.